.github/workflows/code-review.yml

```diff
@@ -178,14 +178,21 @@ jobs:
           CODEBASE:
           {codebase_content}
 
-          Please provide:
-          1. A summary of the changes
-          2. Potential issues or bugs you notice
-          3. Suggestions for improvements
-          4. Any security concerns
-          5. Code quality observations
+          Please provide a brief review in the following format:
 
-          Consider how the changes fit into the existing codebase and if they maintain consistency with the existing code patterns.
+          ## Summary
+          [2-3 sentences maximum describing the main changes]
+
+          ## Key Points to Review
+          - List 2-3 specific areas that need careful attention
+          - Focus on potential issues that could impact functionality or maintainability
+          - Include any security considerations (only if there are any)
```

Comment:
Nice 👍 wondering if we can ask him for potential suggestions for improvement if the second point is found.

As in, from what I understand, it will list potential issues as key points to review, but has no instructions to provide a suggestion on how to fix those potential issues. I guess it does not harm to ask him to provide that, right?